src/tasks/installers/olm.ts

```diff
@@ -80,11 +92,17 @@ export class OLMTasks {
             task.title = `${task.title}...It already exists.`
           } else {
             let subscription: Subscription
-            if (!flags['catalog-source-yaml'] && !flags['catalog-source-name']) {
+
+            // stable Che CatalogSource
+            if (!flags['catalog-source-yaml'] && !flags['catalog-source-name'] && isStableVersion(flags)) {
```

Comment:
We often use `!flags['catalog-source-yaml'] && !flags['catalog-source-name'] && isStableVersion(flags)`
consider moving this block to function.